client/my-sites/menus/menu-editable-item.jsx

```diff
@@ -164,6 +170,10 @@ var MenuEditableItem = React.createClass( {
 				type_family: { $set: this.state.itemType.family }
 			}
 		} ) );
+
+		if ( isInjectedNewPageItem( content ) && !this.state.userChangedName && this.isNew() ) {
+			this.refs.menuLabel.focus();
```

Comment:
I totally agree, but that method is passed waaaay down to the children chain and dispersed between many different children.
And children bind it to onclick
I will maybe protect it by `isMounted` and check if we actually have proper ref